Application/In/Posts/2015-02-27_xCal_xCard_and_RFC_errata.xyl

```diff
@@ -53,7 +52,7 @@ UID:4088E990AD89CB3DBB484909
 END:VEVENT
 END:VCALENDAR</code></pre>
     <p>Horrible, yes. You were warned. These formats are standardized in several
-    RFC, to list some of them:
+    RFCs, to list some of them:
```

Comment:
Yes, definitely. Some people use an apostrophe (RFC's) but I think RFCs is more correct.